internal/client/validate.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package client
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+)
+
+// validate is a helper function to verify the input provided.
+func validate(address, token string) error {
```

Comment:
This is where we add the new `validate()` function.

The purpose of this function is to verify the Vela server address and token are not empty as they are required to communicate with the Vela APIs.